mentoring/participants/models.py

```diff
@@ -23,29 +23,23 @@ class Participant(models.Model):
     A Participant in the program.
     """
 
-    MENTOR = 'M'
-    LEARNER = 'L'
-
     def __str__(self):
-        return f'{self.full_name} ({self.role})'
+        return f'{self.full_name}'
 
     expires = models.DateTimeField(null=False, help_text=dedent('''\
         The date that this information expires.  This can be extended (such as when
         a pairing is made), and expiration is contingent on not being in a current
         pair.  This field accomplishes the "lean data" practice of not keeping
         user information forever. '''))
 
-    email = models.EmailField(null=False, help_text=dedent('''\
+    email = models.EmailField(null=False, unique=True, help_text=dedent('''\
         The participant's work email address.  This is used as a key. '''))
 
-    role = models.CharField(
-        max_length=1,
-        null=False,
-        choices=[(MENTOR, 'Mentor'), (LEARNER, 'Learner')],
-        help_text=dedent('''\
-            The participant's role in the program.  Note that the same email may appear
-            at most once in each role.'''),
-    )
+    is_mentor = models.BooleanField(null=False, help_text=dedent('''\
```

Comment:
Is ``dedent`` used for consistency? This seems like it could just be on a new line.

Reply:
Yeah, it's for consistency, since they tend to get line-wrapped between 1 and >1 lines as they are edited.